docs/pages/access-controls/access-request-plugins/ssh-approval-email.mdx

```diff
@@ -0,0 +1,360 @@
+
+</TabItem>
+</Tabs>
+
+#### `[delivery]`
+
+Assign `sender` to the email address from which you would like the Teleport
+plugin to send messages.
+
+#### `[role_to_recipients]`
+
+The `role_to_recipients` map configure the recipients that the email plugin
+will notify when a user requests access to a specific role. When the plugin
+receives an Access Request from the Auth Service, it will look up the role being
+requested and identify the recipients to notify.
+
+Here is an example of a `role_to_recipients` map:
+
+```toml
+[role_to_recipients]
+"*" = ["security@example.com", "executive-team@example.com"]
+"dev" = "eng@example.com"
+"dba" = "mallory@example.com"
+```
+
+In the `role_to_recipients` map, each key is the name of a Teleport role. Each
+value configures the recipients the plugin will email when it recieves an Access
+Request for that role. The value can be a single string or an array of strings.
+Each string must be an email address.
+
+The `role_to_recipients` map must also include an entry for `"*"`, which the
+plugin looks up if no other entry matches a given role name. In the example
+above, requests for roles aside from `dev` and `dba` will notify
+`security@example.com` and `executive-team@example.com`.
+
+<Details title="Suggested reviewers">
+
+Users can suggest reviewers when they create an Access Request, e.g.,:
+
+```code
+$ tsh request create --roles=dbadmin --reviewers=alice@example.com,ivan@example.com
+```
+If an Access Request includes suggested reviewers, the email plugin will add
+these to the list of recipients to notify. If a suggested reviewer is an email
+address, the plugin will send a message to that recipient in addition to those
+configured in `role_to_recipients`.
+
+</Details>
+
+Configure the email plugin to notify you when a user requests the `editor` role
+by adding the following to your `role_to_recipients` config, replacing
+`YOUR_EMAIL_ADDRESS` with the appropriate address: 
+
+```toml
+[role_to_recipients]
+"*" = "YOUR_EMAIL_ADDRESS"
+"editor" = "YOUR_EMAIL_ADDRESS"
+```
+
+<Details title="Configuring recipients without role mapping">
+
+If you do not plan to use role-to-recipient mapping, you can configure the
+Teleport email plugin to notify a static list of recipients for every Access
+Request event by using the `delivery.recipients` field:
+
+```toml
+[delivery]
+recipients = ["eng@exmaple.com", "dev@example.com"]
+```
+
+If you use `delivery.recipients`, you must remove the `role_to_recipients`
+configuration section. Behind the scenes, `delivery.recipients` assigns the
+recipient list to a `role_to_recipients` mapping under the wildcard value `"*"`.
+
+</Details>
+
+You configuration should resemble the following:
+
+```toml
+# /etc/teleport-email.toml
+[teleport]
+addr = "example.com:3025"
+identity = "/var/lib/teleport/plugins/email/auth_id"
+
+[mailgun]
+domain = "sandboxbd81caddef744a69be0e5b544ab0c3bd.mailgun.org" 
+private_key = "xoxb-11xx"                                     
```

Comment:
We've had some researchers mistake this for a leaked credential and file reports. In those cases, we've changed it to something that is more obviously fake.

```suggestion
private_key = "xoxb-fakekey62b0eac53565a38c8cc0316f6"
```